conf/dynaconf_hooks.py

```diff
@@ -0,0 +1,121 @@
+import json
+
+from robottelo.logging import logger
+from robottelo.utils.ohsnap import dogfood_repository
+
+# from robottelo import constants  # put filename in the constants? TODO
+
+SETTINGS_CACHE_FILE = 'settings_cache-{sat_version}.json'
```

Comment:
If you make this a pathlib object, then it will make writing/reading much easier later.

Reply:
Done, can you check the implementation?